### DISCUSSION TITLE
seeds not randomizing on batch inpainting?

### DISCUSSION THREAD

**Discussion by wmnwmn67** [vladmandic/sdnext#3601] [General]
This must be user error, but when I do inpainting using batch > 1, all the results come out the same. This is with the default -1 seed settings...or any others I have tried. On txt2img I don't have this problem, so I'm not completely clueless...

**Comment by vladmandic** [comment]
i cant reproduce - inpaint with batch size > 1 produces different results. how different depends on denoising strength.
![image](https://github.com/user-attachments/assets/bb99e2bf-038c-45bf-92c6-01b5ddbb174a)

**Reply by wmnwmn67** [reply]
Ok thanks for looking at it! I will try to figure out what I'm doing wrong.
